### PR TITLE
[Keyboard] Add Ciel65

### PR DESCRIPTION
**Pull request by chickenman112**
Add Ciel65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chickenman/ciel65/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+A simple 65% keyboard based of the Bakeneko 65, originially made by Katano.
+
+* Keyboard Maintainer: [ChickenMan](https://github.com/chickenman112)
+* Hardware Supported: Ciel65 HS and Solder pcb
+* Hardware Availability: [To be opensourced](https://github.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make chickenman/ciel65/rev3:default
```
Comment (zhol0777) [inline keyboards/chickenman/ciel65/readme.md:11]:
```suggestion
    qmk compile -kb chickenman/ciel65 -km default
```
?

File: keyboards/chickenman/ciel65/info.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        "rows": ["E6", "B7", "F7", "F4", "F5"],
+        "cols": ["F6", "B0", "F1", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2", "D1", "D0"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": true,
```
Comment (MaiTheSan) [inline keyboards/chickenman/ciel65/info.json]: "command": false,

File: keyboards/chickenman/ciel65/keymaps/default/readme.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# The default keymap for Ciel65 Hotswap and Solder
```
Comment (waffle87) [inline keyboards/chickenman/ciel65/keymaps/default/readme.md:1]: This file should be removed.

File: keyboards/chickenman/ciel65/readme.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Flashing example for this keyboard:
+
+    make chickenman/ciel65/rev3:default:flash
```
Comment (waffle87) [inline keyboards/chickenman/ciel65/readme.md:15]:
```suggestion
    make chickenman/ciel65:default

Flashing example for this keyboard:

    make chickenman/ciel65:default:flash
```

File: keyboards/chickenman/ciel65/readme.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (waffle87) [inline keyboards/chickenman/ciel65/readme.md:17]: Please add instructions to enter bootloader mode. (see [template readme](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md))

**Review by waffle87 — CHANGES_REQUESTED** [review]
`keyboards/chickenman/ciel65/rules.mk` should match [`data/templates/keyboard/rules.mk`](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/rules.mk) too.

File: keyboards/chickenman/ciel65/readme.md (hunk 5 above)
Comment (chickenman112) [reply keyboards/chickenman/ciel65/readme.md:17]: Changed and added za stuffs

Let me know if there are any further issues

**Review by MaiTheSan — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

File: keyboards/chickenman/ciel65/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Ciel65
+
+![Ciel65](https://imgur.com/gallery/16FB1gN)
```
Comment (drashna) [inline keyboards/chickenman/ciel65/readme.md:3]: uummmm...

Comment (chickenman112) [reply keyboards/chickenman/ciel65/readme.md:3]: Fixed my meow-stake

File: keyboards/chickenman/ciel65/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Ciel65
+
+![Ciel65](https://imgur.com/a/8YWNVwb)
```
Comment (MaiTheSan) [inline keyboards/chickenman/ciel65/readme.md:3]:
```suggestion
![Ciel65](https://i.imgur.com/CSOaQRph.jpeg)
```

**Review by fauxpark — APPROVED** [review]

**Pull request merged**